telegram/_bot.py

```diff
@@ -2037,16 +2051,40 @@ async def send_media_group(
                 :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
             api_kwargs (:obj:`dict`, optional): Arbitrary keyword arguments to be passed to the
                 Telegram API.
+            group_caption (:obj:`str`, optional): Text of the caption
+                that will be shown for the whole media group.
+                Defaults to :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
+            group_caption_parse_mode (:obj:`str` | :obj:`None`, optional):
+                Parse mode of the caption that will be shown for the whole media group.
```

Comment:
```suggestion
                Parse mode for :paramref:`group_caption`.
```
similar for `group_caption_entities`

Reply:
yep 👍🏻